src/pages/DashboardPage.js

```diff
@@ -6,31 +6,43 @@
  */
 
 import React from 'react';
-import { View, StyleSheet } from 'react-native';
+import { View, StyleSheet, Text } from 'react-native';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import { ReportSideBar } from '../widgets/ReportSideBar';
 import { ReportChart } from '../widgets/ReportChart';
 
 import globalStyles from '../globalStyles';
+
 import { DashboardActions } from '../actions/index';
 
 const DashboardPageComponent = ({ reports, currentReport, switchReport }) => {
   const { pageContentContainer, container, pageTopSectionContainer } = globalStyles;
   const pageContainer = StyleSheet.flatten([pageTopSectionContainer, { paddingHorizontal: 0 }]);
 
+  if (reports.length) {
+    return (
+      <View style={pageContentContainer}>
+        <View style={container}>
+          <View style={pageContainer}>
+            <ReportSideBar
+              reports={reports}
+              onPressItem={switchReport}
+              currentReport={currentReport}
+              dimensions={localStyles.sidebarDimensions}
+            />
+            <ReportChart report={currentReport} />
+          </View>
+        </View>
+      </View>
+    );
+  }
   return (
     <View style={pageContentContainer}>
       <View style={container}>
-        <View style={pageContainer}>
-          <ReportSideBar
-            reports={reports}
-            onPressItem={switchReport}
-            currentReport={currentReport}
-            dimensions={localStyles.sidebarDimensions}
-          />
-          <ReportChart report={currentReport} />
+        <View style={localStyles.centeredText}>
+          <Text style={localStyles.noReportsText}>No Reports to show</Text>
```

Comment:
localized string please!

Reply:
I couldn't miss that... that's my area haha
Done here c8aa460 and here 996224a